src/WebJobs.Script/Scale/FunctionsScaleManager.cs

```diff
@@ -84,38 +92,36 @@ public virtual async Task<ScaleStatusResult> GetScaleStatusAsync(ScaleStatusCont
                 // this can happen if the host is offline
             }
 
-            var vote = GetAggregateScaleVote(votes, context, _logger);
+            var aggregatedVote = GetAggregateScaleVote(votes, context, _logger);
 
-            return new ScaleStatusResult
-            {
-                Vote = vote
-            };
+            return aggregatedVote;
         }
 
-        internal static ScaleVote GetAggregateScaleVote(List<ScaleVote> votes, ScaleStatusContext context, ILogger logger)
+        internal static int GetAggregateScaleVote(List<int> votes, ScaleStatusContext context, ILogger logger)
         {
-            ScaleVote vote = ScaleVote.None;
+            int vote = 0;
+
             if (votes.Any())
             {
                 // aggregate all the votes into a single vote
-                if (votes.Any(p => p == ScaleVote.ScaleOut))
+                if (votes.Any(p => p > 0))
                 {
                     // scale out if at least 1 monitor requires it
                     logger.LogDebug("Scaling out based on votes");
-                    vote = ScaleVote.ScaleOut;
+                    vote = votes.Max(); // scale out on max vote
                 }
-                else if (context.WorkerCount > 0 && votes.All(p => p == ScaleVote.ScaleIn))
+                else if (context.WorkerCount > 0 && votes.All(p => p < 0))
                 {
                     // scale in only if all monitors vote scale in
                     logger.LogDebug("Scaling in based on votes");
-                    vote = ScaleVote.ScaleIn;
+                    vote = votes.Where(x => x != 0).Max(); // scale in on max vote
```

Comment:
Can the `.Where(x => x != 0)` be removed? The `else if` statement above has a restriction of all votes are less that 0, so no vote here will be 0. Or does `voles.All(p => p < 0)` need to change instead?